tests/install_upgrade_operators/utils.py

```
@@ -299,23 +322,32 @@ def apply_konflux_idms(
+            return
+
     with ResourceEditor(patches={mcp: {"spec": {"paused": True}} for mcp in machine_config_pools}):
         if idms.exists:
-            LOGGER.info(f"Patching IDMS {KONFLUX_IDMS_NAME} with mirrors: {required_mirrors}")
-            ResourceEditor(patches={idms: {"spec": {"imageDigestMirrors": image_digest_mirrors}}}).update()
+            LOGGER.info(f"Patching IDMS {idms.name} with missing mirrors for: {required_mirrors}")
+            ResourceEditor(patches={idms: {"spec": {"imageDigestMirrors": updated_entries}}}).update()
         else:
-            LOGGER.info(f"Creating IDMS {KONFLUX_IDMS_NAME} with mirrors: {required_mirrors}")
+            image_digest_mirrors = [{"source": RH_IDMS_SOURCE, "mirrors": required_mirrors}]
+            LOGGER.info(f"Creating IDMS {idms.name} with mirrors: {required_mirrors}")
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)